ci_feature/main.sh

```diff
@@ -196,16 +174,94 @@ if [ "$cAdvisorIsSecure" = true ] ; then
       echo "export IS_SECURE_CADVISOR_PORT=true" >> ~/.bashrc
       export CADVISOR_METRICS_URL="https://$NODE_IP:10250/metrics"
       echo "export CADVISOR_METRICS_URL=https://$NODE_IP:10250/metrics" >> ~/.bashrc
+      echo "Making wget request to cadvisor endpoint /pods with port 10250 to get the configured container runtime on kubelet"
+      podsResponse=$(wget -O- --server-response https://$NODE_IP:10250/pods --no-check-certificate --header="Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)")
+      if [ -z "$podsResponse" ]; then
+            echo "-e error  wget request to cadvisor endpoint /pods with port 10250 to get the configured container runtime on kubelet failed"
+            # should we default to container runtime docker?
+      else
+            containerRuntime=$(echo $podsResponse | jq -r '.items[0].status.containerStatuses[0].containerID' | cut -d ':' -f 1)
+            nodeName=$(echo $podsResponse | jq -r '.items[0].spec.nodeName')
```

Comment:
can you do a count before accessing [0] ?

Reply:
yep. let me do more checks.